OpenKeychain/src/main/res/values/themes.xml

```diff
@@ -3,7 +3,7 @@
 
     <style name="KeychainTheme" parent="KeychainTheme.Base" />
 
-    <style name="KeychainTheme.Base" parent="Theme.AppCompat.Light">
+    <style name="KeychainTheme.Base" parent="MaterialDrawerTheme.Light.ActionBar">
```

Comment:
No, no. I already removed that, thinking it was that. But even with `Theme.AppCompat.Light` it changes the color. Very strange. I'll look into it.